### PR TITLE
Trusty: Fix the motd reset

### PR DESCRIPTION
**Pull request by andyzheng0831**
I made a couple of fixes about GCI motd in the master  and release-1.3 branches. But while I made a cherrypick, there was much conflict, and manually solving the conflict apparently made a mistake of omitting something. This PR is for fixing such a problem. Please note that this is the minimal change to make GCI in release-1.2 green.

@zmerlynn please help me set the milestone. The v1.2 is not visible to me now. Thanks!

### CODE REVIEW

File: cluster/gce/trusty/master.yaml [hunk 1]
```diff
@@ -213,6 +213,7 @@ script
 	set -o errexit
 	set -o nounset
 
+	. /etc/kube-configure-helper.sh
```
Comment (roberthbailey) [inline cluster/gce/trusty/master.yaml]: On the master branch, there is a sleep 60 before this line w/ a TODO. Should that be backported?

File: cluster/gce/trusty/configure-helper.sh [hunk 2]
```diff
@@ -726,7 +726,7 @@ Note: This looks like a development version, which might not be present on GitHu
 If it isn't, the closest tag is at:
   https://github.com/kubernetes/kubernetes/tree/${gitref}
 "
-    gitref="${version//*+/}"
+    gitref=$(echo ${version} | sed -e 's/.*+//')
```
Comment (zmerlynn) [inline cluster/gce/trusty/configure-helper.sh]: Why is this needed? And why the change versus `head` (where it looks like the left in both gci and trusty)?

```
$ version=1.4.0-alpha.0.210+6209b1b60cd0d9
$ echo "${version//*+/}"
6209b1b60cd0d9
$ echo $(echo ${version} | sed -e 's/.*+//')
6209b1b60cd0d9
```

File: cluster/gce/trusty/master.yaml (hunk 1 above)
Comment (andyzheng0831) [inline cluster/gce/trusty/master.yaml]: I think what you mentioned is the health monitoring job, which is different from here.

File: cluster/gce/trusty/configure-helper.sh (hunk 2 above)
Comment (andyzheng0831) [inline cluster/gce/trusty/configure-helper.sh]: The line "gitref="${version//*+/}"" does not work in upstart job, but works in shell script. The GCI config executes a shell script via systemd, but trusty config directly calls shell functions in upstart jobs so it does not work.

This is a problem I once mentioned in the discussion of maintenance plan for trusty code. We decided not to guarantee it green in the master branch to reduce the maintenance cost. As a result, there have been several key fixes merged in GCI config are not made for trusty config, and trusty config is broken. The downside is that it is hard to verify if a change is 100% correct. Here is one case of such situation. I noticed that gitref="${version//*+/}" does not work, but apparently I did not make a change in the master branch or release-1.3. And later in 1.2 branch I saw the failure and so I made this PR to ensure it is green in release-1.2 branch.

Comment (zmerlynn) [inline cluster/gce/trusty/configure-helper.sh]: So this actually does belong as a cherrypick from master?

File: cluster/gce/trusty/master.yaml (hunk 1 above)
Comment (roberthbailey) [inline cluster/gce/trusty/master.yaml]: Correct. I guess this has diverged significantly from master, since I don't see a block on the master branch with `filename="kube-addons.conf"`.

File: cluster/gce/trusty/configure-helper.sh (hunk 2 above)
Comment (andyzheng0831) [inline cluster/gce/trusty/configure-helper.sh]: Not really. It is more like a fix to a previous cherrypick. The master and release-1.3 branch trusty code diverges from release-1.2 too much. Manual work was heavily involved in recent cherrypicks to release-1.2 and I made a mistake there. So I would like to fix it in this PR

File: cluster/gce/trusty/master.yaml (hunk 1 above)
Comment (andyzheng0831) [inline cluster/gce/trusty/master.yaml]: Yes, kube-addons were moved into a static pod kube-addon manager since release-1.3, but in release-1.2 they are still managed by the old fashion scripts. This should be the reason why you see they are quite different.

File: cluster/gce/trusty/configure-helper.sh (hunk 2 above)
Comment (zmerlynn) [inline cluster/gce/trusty/configure-helper.sh]: Sorry, I'm confused. Is this working on `master`?

Comment (andyzheng0831) [inline cluster/gce/trusty/configure-helper.sh]: For master branch, the GCI code works fine and we have plenty of Jenkins jobs covering it. For trusty code, no, it was broken since one month ago. Even we fix this line in the trusty code in master branch, it does not work. We don't have any auto test covering trusty config in master branch, and we are not trying to making it green.
